src/psyclone/tests/psyir/frontend/fparser2_derived_type_test.py

```diff
@@ -278,14 +330,28 @@ def test_derived_type_ref(f2008_parser):
     assert isinstance(amem, ArrayOfStructuresMember)
     assert isinstance(amem.member, ArrayMember)
     assert isinstance(amem.member.indices[0], Range)
-    assign = assignments[4]
+    assert isinstance(amem.member.indices[0].children[0], BinaryOperation)
+    assert amem.member.indices[0].children[0].children[0].symbol.name == "var"
+    gen = fortran_writer(amem)
+    assert (gen == "subgrid(3)%data(:)")
```

Comment:
pylint says the parenthesis are not needed

Reply:
Done.